content/blog/arduino-esp-now-lib/index.md

```diff
@@ -0,0 +1,456 @@
+---
+title: "Using ESP-NOW in Arduino"
+date: 2024-06-03T09:00:24-03:00
+tags: ["Arduino", "ESP-NOW", "Wireless", "Wi-Fi", "ESP32", "ESP32-S2", "ESP32-S3", "ESP32-C3", "ESP32-C6"]
+showAuthor: false
+authors:
+  - "lucas-vaz"
+---
+
+ESP-NOW is a connectionless Wi-Fi communication protocol developed by Espressif for its microcontrollers. It allows for efficient, low-power, and low-latency peer-to-peer communication. This article will guide you through the essentials of using the new ESP-NOW library included in Arduino 3.0.0, from basic setup to advanced features.
```

Comment:
It might me useful to include the link to the product page.

https://www.espressif.com/en/solutions/low-power-solutions/esp-now